python_http_client-3.3.3.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
python_http_client-3.3.3.dist-info/LICENSE,sha256=DAmPnLpzLJRaOr1asDSBug2-cxQIrKxPXfsCWh2qe24,1097
python_http_client-3.3.3.dist-info/METADATA,sha256=W--qoUnQsqQj7lnTY4Ins24eSYXZaPtCaRY0IZvxC-A,6715
python_http_client-3.3.3.dist-info/RECORD,,
python_http_client-3.3.3.dist-info/REQUESTED,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
python_http_client-3.3.3.dist-info/WHEEL,sha256=ewwEueio1C2XeHTvT17n8dZUJgOvyCWCt0WVNLClP9o,92
python_http_client-3.3.3.dist-info/top_level.txt,sha256=2JnUimcxRHkpXlcL22LcQStBT2-2f7uE6bN32da2nX0,19
python_http_client/VERSION.txt,sha256=79m7Mo62nynenbuHAHP-j0FuPryg0QAl3SOzzihV5qY,6
python_http_client/__init__.py,sha256=fU3Hvupc6zEfI7tHmwTzPq5d6M_9SJ-zc455ROEa38E,604
python_http_client/__pycache__/__init__.cpython-310.pyc,,
python_http_client/__pycache__/client.cpython-310.pyc,,
python_http_client/__pycache__/exceptions.cpython-310.pyc,,
python_http_client/client.py,sha256=dkdI9Y86GzKmH3F-XRe8G2vWOXyB8erS3IaQuZpVJM0,9921
python_http_client/exceptions.py,sha256=cP9hcA1kZMIrdHleSGcaoDyeJRX7bOFt_VCds-WV2Tk,1796
